File: Basic/Check Whether a Number is Even or Odd/SolutionByGurirath.cpp
#include <bits/stdc++.h>
using namespace std;
void checkevenOdd(int n)
{
    if (n % 2 == 0) {

        cout << "Even";
    } else {
        cout << "Odd";
    }
}
// Main function
int main()
{

    int n;
    cout << "enter the number : ";
    cin >> n;
    checkevenOdd(n);
    cout << endl;

    return 0;
}

File: Basic/Check Whether a Number is Even or Odd/SolutionByGurirath.java

/**
 Write a number to check if it is even or odd.
 */
import java.util.*;
public class hacktoberfest2
{
  public static void main(String args[])
  {
      Scanner kb= new Scanner(System.in);
      int n;
      System.out.println("Enter a number to check if it is even or odd");
      n=kb.nextInt();
      if(n%2==0)
      {
          System.out.println("The number is even");
      }
      else
      {
          System.out.println("The number is odd");
      }
  }
}
